pyproject.toml

```diff
@@ -28,4 +28,4 @@ use_parentheses = true
 skip_glob = ["build/*", "stubs/*"]
 # Don't split "import" and "from".
 force_sort_within_sections = true
-known_third_party = ["benchmark_dataset", "dataclasses", "datasets", "golden_configs", "helpers", "models", "numpy", "pytest", "recommonmark", "setuptools", "torch", "torch_pg", "torchtext", "torchvision"]
```

Comment:
I've made the same change after @msbaines 's comment in PR:https://github.com/facebookresearch/fairscale/pull/348.

Reply:
this is not enough, the pre-commit file also needs to be fixed, that's what I'm trying to say. This file is updated automatically